.github/workflows/ci.precheck.yml

```diff
@@ -0,0 +1,24 @@
+# This workflow will check stub files, integrity, and potentially other checks
+
+name: exercism/javascript ci/precheck
+
+on:
+  push:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
```

Comment:
This workflow doesn't contain a strategy matrix. Is that intended? If so, it's probably better to use a specific version here.

Reply:
No, forgot to put it in after removing them matrix. Thanks!